templates/commands/render_action_include.go

```diff
@@ -51,7 +51,7 @@ func actionInclude(ctx context.Context, i *model.Include, sp *stepParams) error
 		// directory will be overwritten; that comes later.
 		const overwrite = true
 
-		if err := copyRecursive(p.Pos, absSrc, absDst, sp.fs, overwrite); err != nil {
+		if err := copyRecursive(p.Pos, absSrc, absDst, sp.fs, overwrite, false); err != nil {
```

Comment:
Why this change? We should overwrite when copying to scratch, right?

Reply:
I think you've mistaken the dryRun param for the overwrite param.

Reply:
Ah, right!